bin/delete_data.py

```diff
@@ -0,0 +1,168 @@
+__author__ = 'mwham'
+import sys
+import os
+from os.path import join
+from datetime import datetime
+import argparse
+import logging
+import requests
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from analysis_driver.exceptions import AnalysisDriverError
+from analysis_driver.config import default as cfg, logging_default as log_cfg
+from analysis_driver.app_logging import AppLogger
+from analysis_driver import rest_communication
+from analysis_driver import executor
+
+
+class Deleter(AppLogger):
+    def __init__(self, work_dir, dry_run=False, deletion_limit=50):
+        self.work_dir = work_dir
+        self.dry_run = dry_run
+        self.deletion_limit = deletion_limit
+
+    def _execute(self, cmd, cluster_execution=False):
+        if self.dry_run:
+            self.info('Running: ' + cmd)
+            status = None
+        elif cluster_execution:
+            status = executor.execute([cmd], job_name='data_deletion', working_dir=self.work_dir).join()
+        else:
+            status = executor.execute([cmd], 'local').join()
+        if status:
+            raise AnalysisDriverError('Command failed: ' + cmd)
+
+    def _compare_lists(self, observed, expected, error_message='List comparison mismatch:'):
+        observed = sorted(observed)
+        expected = sorted(expected)
+        if observed != expected:
+            self.error(error_message)
+            self.error('observed: ' + str(observed))
+            self.error('expected: ' + str(expected))
+
+            if not self.dry_run:
+                raise AssertionError
+
+
+class RawDataDeleter(Deleter):
+    deletable_sub_dirs = ('Data', 'Logs', 'Thumbnail_Images')
+
+    def deletable_runs(self):
+        non_deleted_runs = query_api(
+            'runs',
+            'embedded={"run_elements":1,"analysis_driver_procs":1}',
+            'aggregate=True',
+            'max_results=' + str(self.deletion_limit)
+        )
+
+        deletable_runs = []
+        for r in non_deleted_runs:
+            review_statuses = r.get('review_statuses')
```

Comment:
Has this "review_statuses" aggregation been deployed already?
I can seem to find it in the production server